### ISSUE TITLE
Issue resolving some nightlies

### ISSUE THREAD

**Issue by ckipp01** [lampepfl/dotty#14876]
I can't really figure out what's going on here, but I have a script that is supposed to run every night to grab the newest nightly and then run some code with it. However, last night and today it's failing due to not being able to resolve a scala version that says is published and is located in the `maven-metadata.xml`, but the artifacts aren't there.

According to this [release](https://github.com/lampepfl/dotty/runs/5845050161?check_suite_focus=true), `3.1.3-RC1-bin-20220406-73cda0c-NIGHTLY` was successfully published and according to https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/maven-metadata.xml it should be the newest valid `3.1.3*` version. However, if you look at the listing on [here](https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/), the artifacts aren't there. You can see that it won't resolve with:

```
cs fetch org.scala-lang:scala3-compiler_3:3.1.3-RC1-bin-20220406-73cda0c-NIGHTLY
```

I know this isn't the newest version as `3.1.3-RC1-bin-20220406-3d1d299-NIGHTLY` was published last night, but any idea what's going on here as that artifact should be there?

**Comment by griggt** [comment]
Maven Central had a sync issue / outage: https://status.maven.org/incidents/9t2xfqj6mxqn

Note that they claim to have fixed it and

> Artifacts are now in sync as normal in repo1.maven.org.

But that does not seem to be the case here.

I believe the job to release 3.1.3-RC1 also ran during the outage, and it has not appeared yet either.

**Comment by griggt** [comment]
The artifacts for 3.1.3-RC1 have still not appeared on Maven Central.  Perhaps we need to open a support ticket with Sonatype?

**Comment by Kordyjan** [comment]
I published 3.1.3-RC2 with the same changes (except the things depending on version number) as 3.1.3-RC1. The number we start with doesn't really matter for the RCs.

For the sake of tooling, it might be better if the artifacts for the two missing releases (`3.1.3-RC1` and `3.1.3-RC1-bin-20220406-73cda0c-NIGHTLY`) were synchronized with maven central. Right now it is strange, as they are listed in [the version list](https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/maven-metadata.xml) but there are [no artifacts](https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/).

@anatoliykmetyuk You are more knowledgable than me in terms of how exactly our artifacts are committed to the maven central. Could you please open the issue on the sonatype support Jira?

**Comment by ckipp01** [comment]
Just to be clear, I probably wouldn't recommend spending a lot of time on this. While it's an inconvenience if someone tries to grab those versions, I have a feeling that it will impact very very few people. I was only aware of it because CI in both https://github.com/ckipp01/dotty-error-index and https://github.com/scalameta/metals both failed that night since they both rely on pulling in the latest nightly. Since there are newer versions, it's done with and isn't really an issue. It's up to you if you want to chase it down, but please don't on my account.

**Comment by anatoliykmetyuk** [comment]
I agree with @ckipp01, the nightly version skipped is probably not going to be a problem, as long as we have the RCs version in order.
